README.md

```diff
@@ -28,50 +28,50 @@ This repo aggregates all audits performed on Sablier smart contracts by third-pa
-| Egis Security | Firm    | [`abf7154`][abf7154], [`f9defae`][f9defae] | [2024-06-28](./lockup/v1.2/20240628_egis.pdf)    |
-| CodeHawks     | Contest | [`c85c66a`][c85c66a], [`eb2983d`][eb2983d] | [2024-06-15](./lockup/v1.2/20240615_codehawk.md) |
-| Cantina       | Firm    | [`a86edee`][a86edee], [`73831c7`][73831c7] | [2024-06-13](./lockup/v1.2/20240613_cantina.pdf) |
+| Auditors      | Types   | Commits                                    | Reports                                            |
+| :------------ | :------ | :----------------------------------------- | :------------------------------------------------- |
+| Cantina       | Firm    | [`36b49d3`][36b49d3], [`9d7a15b`][9d7a15b] | [2024-07-03](./lockup/v1.2.0/20240703_cantina.pdf) |
+| Egis Security | Firm    | [`abf7154`][abf7154], [`f9defae`][f9defae] | [2024-06-28](./lockup/v1.2.0/20240628_egis.pdf)    |
+| CodeHawks     | Contest | [`c85c66a`][c85c66a], [`eb2983d`][eb2983d] | [2024-06-15](./lockup/v1.2.0/20240615_codehawk.md) |
+| Cantina       | Firm    | [`a86edee`][a86edee], [`73831c7`][73831c7] | [2024-06-13](./lockup/v1.2.0/20240613_cantina.pdf) |
 
-### v1.1
+### v1.1.0
 
 [e5a227f]: https://github.com/sablier-labs/v2-core/tree/e5a227f77ededaf4d2737b36ed958445ad86eee9
 [0004fd2]: https://github.com/sablier-labs/v2-periphery/tree/0004fd2e61e032df3d895045ec414ecb212ddcc8
 [07014ac]: https://github.com/sablier-labs/v2-core/tree/07014ac
 [cc9434f]: https://github.com/sablier-labs/v2-periphery/tree/cc9434f
 [79d88ca]: https://github.com/sablier-labs/v2-periphery/tree/79d88ca
 
-| Auditors          | Types | Commits                                    | Reports                                          |
-| :---------------- | :---- | :----------------------------------------- | :----------------------------------------------- |
-| Cantina           | Firm  | [`e5a227f`][e5a227f], [`0004fd2`][0004fd2] | [2023-12-21](./lockup/v1.1/20231221_cantina.pdf) |
-| Cantina           | Firm  | [`07014ac`][07014ac], [`cc9434f`][cc9434f] | [2023-12-15](./lockup/v1.1/20231215_cantina.pdf) |
-| Turing Consulting | Firm  | [`e5a227f`][e5a227f], [`0004fd2`][0004fd2] | [2023-11-30](./lockup/v1.1/20231130_turing.pdf)  |
-| Cantina           | Firm  | [`79d88ca`][79d88ca]                       | [2023-09-06](./lockup/v1.1/20230906_cantina.pdf) |
+| Auditors          | Types | Commits                                    | Reports                                            |
+| :---------------- | :---- | :----------------------------------------- | :------------------------------------------------- |
+| Cantina           | Firm  | [`e5a227f`][e5a227f], [`0004fd2`][0004fd2] | [2023-12-21](./lockup/v1.1.0/20231221_cantina.pdf) |
+| Cantina           | Firm  | [`07014ac`][07014ac], [`cc9434f`][cc9434f] | [2023-12-15](./lockup/v1.1.0/20231215_cantina.pdf) |
+| Turing Consulting | Firm  | [`e5a227f`][e5a227f], [`0004fd2`][0004fd2] | [2023-11-30](./lockup/v1.1.0/20231130_turing.pdf)  |
```

Comment:
Calling it `v1.1` is better because some of them refer to `v1.1.0` while some of them refer to `v1.1.2`.

Reply:
similar to `package.json` file, what about: `v1.1.x`